### ISSUE TITLE
Windows 7

### ISSUE THREAD

**Issue by harryray33** [srevinsaju/guiscrcpy#352]
Does this work with Windows 7?

Thanks.

**Comment by srevinsaju** [comment]
nope, but you can use an older release which works with Windows 7. It must be a release from 2020 of 2019

**Comment by harryray33** [comment]
Thanks, what's the last version that works, is 4.80 OK? 
4.8 appears to be the last version in 2020
